streams/src/test/java/org/apache/kafka/streams/tests/SmokeTestUtil.java

```diff
@@ -29,12 +29,8 @@
 import org.apache.kafka.streams.processor.api.ProcessorContext;
 import org.apache.kafka.streams.processor.api.ProcessorSupplier;
 import org.apache.kafka.streams.processor.api.Record;
-import org.junit.Rule;
-import org.junit.rules.Timeout;
 
 public class SmokeTestUtil {
-    @Rule
-    public Timeout globalTimeout = Timeout.seconds(600);
```

Comment:
Had to remove this since system tests do not have a dependency to junit that contains`Timeout`. AFAIS, the timeout rule makes no sense here since this class and all child-classes except `RelationalSmokeTestTest` do not contain tests. I added the timeout rule to `RelationalSmokeTestTest`.